scarpe-components/lib/scarpe/components/calzini/para.rb

```diff
@@ -134,8 +135,12 @@ def contains_number?(str)
         next
       else
         if contains_number?(i)
-          
-          fss=i;
+
+          if contains_only_numbers?(i)
```

Comment:
Interesting. I was going to suggest we might want a more flexible font-size specification, but it looks like Shoes doesn't do that. So you're good here :-)